### PR TITLE
Add shower layer to toilets theme

### PR DESCRIPTION
**Pull request by RobinLinde**
Adds a basic shower layer to the toilets theme with questions for access, fee, payment, opening hours and hot water.

### CODE REVIEW

File: assets/layers/shower/shower.json [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      },
+      "mappings": [
+        {
+          "if": "fee=yes",
+          "then": {
+            "en": "There is a fee for using this shower"
+          }
+        },
+        {
+          "if": "fee=no",
```
Comment (pietervdvn) [inline assets/layers/shower/shower.json:70]: Perhaps add an `addExtraTags: ["charge="]` here? This will remove the `charge`-tag again

**Review by pietervdvn — APPROVED** [review]

**Comment by pietervdvn** [comment]
Excellent work, as usual!

File: assets/layers/shower/shower.json (hunk 1 above)
Comment (RobinLinde) [reply assets/layers/shower/shower.json:70]: Good catch, added

Comment (pietervdvn) [reply assets/layers/shower/shower.json:70]: It's rare for me to spot a possible improvement in your PR's!

**Pull request merged**